packages/asset-swapper/package.json

```diff
@@ -43,6 +43,7 @@
         "@0x/assert": "^2.2.0-beta.2",
         "@0x/contract-addresses": "^3.3.0-beta.4",
         "@0x/abi-gen-wrappers": "^5.4.0-beta.3",
+        "@0x/contracts-erc20-bridge-sampler": "^1.0.0-beta.1",
```

Comment:
TODO add this to `abi-gen-wrappers`/export from `contract-wrappers`. These have pruned artifacts and won't bloat out the asset-swapper bundle size.